canonical-websites/www.ubuntu.com
# Remove pre-vanilla files, rename ‘v1’ files

## Done

* in pages, renamed includes to use non ‘-v1’ files
* in base templates, shared files, deleted old versions and renamed v1 versions

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- look at site to see if it still renders correctly
